### PR TITLE
Feature/43692 migrate census-import to typescript

### PR DESCRIPTION
**Pull request by GuyHarwood**
- [x] move to `tslib/throttled-functions`
- [x] refactor to typescript
- [x] leverage benefits 💰 
- [x] improve unit tests 🚀

### CODE REVIEW

**Review by jon-shipley — COMMENTED** [review]
LGSF

File: tslib/src/functions-throttled/census-import/census-import.data.service.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    table.columns.add('gender', mssql.NVarChar(mssql.MAX), { nullable: false })
+    table.columns.add('dob', mssql.NVarChar(mssql.MAX), { nullable: false })
+    // skip zero index entry, as its a header row
+    for (let i = 1; i < blobContent.length; i++) {
+      const row = blobContent[i]
+      table.rows.add(i, row[0], row[1], row[2], row[3], row[4], row[5], row[6], row[7], row[8])
+    }
+    const request = new mssql.Request(this.pool)
+    const result = await request.bulk(table)
+    console.log(`bulk result:${JSON.stringify(result, null, 2)}`)
```
Comment (jon-shipley) [inline tslib/src/functions-throttled/census-import/census-import.data.service.ts:43]: Are we keeping this log line in?

**Review by jon-shipley — APPROVED** [review]
LGTM 👍

File: tslib/src/functions-throttled/census-import/census-import.data.service.ts (hunk 1 above)
Comment (GuyHarwood) [reply tslib/src/functions-throttled/census-import/census-import.data.service.ts]: fixed

**Pull request merged**